easybuild/easyconfigs/s/SMRT-Link/SMRT-Link-5.1.0.26412-cli-tools-only.eb

```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'Binary'
+
+name = 'SMRT-Link'
+version = '5.1.0.26412'
+versionsuffix = '-cli-tools-only'
+
+homepage = 'https://www.pacb.com/support/software-downloads/'
+description = """PacBio’s open-source SMRT Analysis software suite is designed for use with Single Molecule,
+ Real-Time (SMRT) Sequencing data. You can analyze, visualize, and manage your data through an intuitive GUI
+ or command-line interface. You can also integrate SMRT Analysis in your existing data workflow through
+ the extensive set of APIs provided"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# The download is a password protected zip file
+# Download it from https://www.pacb.com/support/software-downloads/ and uncompress
+# the .run file to your sources folder
+sources = ['smrtlink_%(version)s.run']
+checksums = ['629c42b14b2ba9982f0a21d75ffb3cfccb7e43bac0927d2239a6e7b08b89c67d']
+
+install_cmd = '%(builddir)s/smrtlink_%(version)s.run --rootdir %(installdir)s --smrttools-only'
+
+cleanupoldinstall = True
+dontcreateinstalldir = True
+
+modextravars = {
+    'SMRT_ROOT': '%(installdir)s',
+}
+
+modextrapaths = {
+    'PATH': ['smrtcmds/bin', 'smrtcmds/developer/bin', 'admin/bin'],
+}
+
+sanity_check_paths = {
+    'files': [],
```

Comment:
include a couple of key binaries? you're now not checking whether the location you add to `$PATH` actually exist